packages/driver/src/multi-domain/communicator.ts

```diff
@@ -8,11 +8,11 @@ const debug = debugFn('cypress:driver:multi-domain')
 
 const CROSS_DOMAIN_PREFIX = 'cross:domain:'
 
-const serializeForPostMessage = (value) => {
+const preprocessErrorForPostMessage = (value) => {
   const { isDom } = $dom
 
   if (_.isError(value)) {
-    const serializedError = _.mapValues(clone(value), serializeForPostMessage)
+    const serializedError = _.mapValues(clone(value), preprocessErrorForPostMessage)
```

Comment:
Do we still want to call this a `serializedError`? I was almost thinking `processedError` but that almost sounds more confusing 😅 
```suggestion
    const processedError = _.mapValues(clone(value), preprocessErrorForPostMessage)
```

Reply:
Also, if we are using this for only errors now and not generic serialization, I wonder if we can gut like 90% of this function and just imply that value is of type `Error` or if its `undefined` just do nothing?